GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +68,428 @@ Mesh* AssimpImporter::getMesh(int index) {
 
     return mesh;
 }
+
+bool AssimpImporter::create_instance(JNIEnv *env, const char* class_name,
+        jobject& new_instance) {
```

Comment:
IMHO the JNI logic should be in *_jni.cpp file. I can see in the framework all the JNI related code is restricted to *_jni.cpp files only.